docs/completeExamples/completeExamples.xml

```diff
@@ -0,0 +1,485 @@
+                </Source>
+                <Destination>
+                    <Device>
+                        <HostName>yourhost.yourdomain.com</HostName>
+                    </Device>
+                </Destination>
+                <Payload>
+                    <File>
+                        <Size>12345321</Size>
+                        <Hash>efd1dffd90296a69a8aecd7ecb1832b7</Hash>
+                        <Type>text/xml</Type>
+                        <Path>/appdata/alldata/gooddata/bestdata.xml</Path>
+                    </File>
+                </Payload>
+            </Send>
+        </EventDetail>
+    </Event>
+    <Event>
+        <EventTime>
+            <TimeCreated>2017-01-02T03:04:05.678Z</TimeCreated>
+        </EventTime>
+        <EventSource>
+            <System>
+                <Name>Rock Sample Database</Name>
+                <Environment>Space</Environment>
+                <Organisation>ACMECoolResearch</Organisation>
+                <Version>R8.1</Version>
+            </System>
+            <Generator>db-query</Generator>
+            <Device>
+                <HostName>db56.serverfarm.mydomain.org</HostName>
+                <IPAddress>191.181.171.161</IPAddress>
+            </Device>
+            <Client>
+                <HostName>desktop4.moonbase-a.mydomain.org</HostName>
+                <IPAddress>111.101.101.111</IPAddress>
+            </Client>
+            <User>
+                <Id>jc101</Id>
+                <UserDetails>
+                    <Id>jc101</Id>
+                    <StaffNumber>101</StaffNumber>
+                    <Surname>Coder</Surname>
+                    <Initials>JD</Initials>
+                    <Title>Mr</Title>
+                    <KnownAs>Jolly</KnownAs>
+                    <PersonType>Employee</PersonType>
+                    <Group>Technology</Group>
+                    <Unit>Geology</Unit>
+                    <Position>T.SD/42</Position>
+                    <Role>Geologist</Role>
+                    <GradeOfPost>E.5</GradeOfPost>
+                    <EmploymentType>Employee</EmploymentType>
+                    <EmploymentStatus>Active</EmploymentStatus>
+                    <Nationality>GB</Nationality>
+                    <Location>Moon Base Alpha</Location>
+                    <RoomNumber>4/31</RoomNumber>
+                    <Phone>11223</Phone>
+                    <SupervisorStaffNumber>421</SupervisorStaffNumber>
+                    <Organisation>ACMECoolResearch</Organisation>
+                    <HostOrganisation>MoonCo</HostOrganisation>
+                </UserDetails>
+            </User>
+            <Interactive>true</Interactive>
+        </EventSource>
+        <EventDetail>
+            <TypeId>findByConstraint</TypeId>
+            <Description>User has queried database using specified constraints</Description>
+            <Search>
+                <Query>
+                    <Raw>type=r, size=large, colour=white</Raw>
```

Comment:
I would show a whole sql select for a raw, thus implies that the schema takes the actual query command that was run.